chapter_appendix-mathematics-for-deep-learning/single-variable-calculus_vn.md

```diff
@@ -220,10 +220,10 @@ Thus we may conclude, correctly, that the value we seek (the degree a change in
 The way that a mathematician encodes this fact is
 -->
 
-Nếu để ý kĩ, chúng ta sẽ nhận ra rằng kết quả của con số này là xấp xỉ $8$.
-Trong trường hợp ta giảm $\epsilon$ thì giá trị đầu ra ngày càng tiến gần đến $8$.
+Nếu để ý kĩ, chúng ta sẽ nhận ra rằng kết quả của con số này xấp xỉ $8$.
+Trong trường hợp ta giảm $\epsilon$ thì giá trị đầu ra ngày càng tiến gần đến $8$. 
 Vì vậy chúng ta có thể kết luận một cách chính xác, rằng giá trị mà chúng ta tìm kiếm (độ biến thiên đầu vào thay đổi đầu ra) là $8$ tại điểm $x=4$.
-Cách mà một nhà toán học giải mã vấn đề trên là:
+Một nhà toán học sẽ mã hóa vấn đề trên như sau:
```

Comment:
```suggestion
Một nhà toán học sẽ viết như sau:
```

Reply:
Câu này chưa ổn, mình nghĩ nên bỏ chữ nhà toán học đi luôn
```suggestion
Vấn đề trên được viết dưới dạng toán học như sau:
```

Reply:
```suggestion
Có thể viết dưới dạng toán học như sau:
```
mình thấy từ `vấn đề` không đúng lắm, thế này thì sao?

Reply:
Yeah, v đc hơn